plugins/module_utils/system_patch.py

```diff
@@ -0,0 +1,232 @@
+from base64 import b64encode
+import os
+
+from ansible.module_utils._text import to_text
+
+from ansible_collections.pfsensible.core.plugins.module_utils.module_base import PFSenseModuleBase
+
+# <pfsense>
```

Comment:
Do we really want these comments?

Reply:
was for the developing usefull - i delete it